.gitignore

```diff
@@ -133,3 +133,4 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+Surfshark.png
```

Comment:
I would prefer that you add it into your own ignore file instead of the project gitignore because you're the only one to have this file. If you need more information about how to do it you can check this : [https://git-scm.com/docs/gitignore](https://git-scm.com/docs/gitignore) it should be in surfshark-linux-client/.git/info/exclude

Reply:
Thanks for your feedback. I didn't knew that the keyring would ask for the password in the terminal on some distros. On my machine it opens a  normal password window and would leave me the option to never ask to unlock it again. But when that's something not every distro does it's definitely better to keep the old system. I removed the part of the .gitignore and will put it in my local exclude list.

Reply:
I just changed an important thing in the enablekillswitch.sh I don't know why but I wrote ACCEPT IPv6 output which is definitely nothing the killswitch should do.

Reply:
Yes I use Arch with MATE and I need to use the terminal, I guess that's on gnome you have a popup.
Okay, we can keep the idea of keyrings though if we find a better solution ;)
Thanks for the gitignore :D

Reply:
Okay, I don't know that much about ipatables so, as I said, I need to check few things but it should be that long ;)

Reply:
So can you also remove the changes you done for the log in ? :D

Reply:
Yeah sure. Give me a moment :)

Reply:
Ok. Should be reverted now

Reply:
I reviewed the last commit ;)
Also if you could revert the changes on the Readme.md ? that would be perfect

Reply:
And take the time you need ;)